src/foam/nanos/u2/navigation/FooterView.js

```diff
@@ -82,6 +84,11 @@ foam.CLASS({
     ^ .mini-links {
       float: left;
     }
+    ^ button {
+      border: none !important;
```

Comment:
Is there a way to do this without `!important` ?  Using !important makes it harder to override later, as there's not `!evenMoreImportant` in CSS.

Reply:
I added a fourth `ButtonStyle` value in the enum: `UNSTYLED`, which can be used in cases like this where we want to use an action for all the features that come with them, but don't want the default button styles. This will let us remove a lot of CSS overrides like this file was doing.